2025-07-09 00:17:57,501 - asyncio - DEBUG - Using proactor: IocpProactor
2025-07-09 00:17:57,501 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-09 00:17:57,508 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-09 00:17:57,517 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-09 00:17:57,517 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-09 00:17:57,526 - __main__ - INFO - Telegram Manager Bot started. Send /menu to interact.
2025-07-09 00:17:57,526 - telegram.ext.ExtBot - DEBUG - Entering: send_message
2025-07-09 00:17:57,541 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-09 00:18:02,556 - httpcore.connection - DEBUG - connect_tcp.failed exception=ConnectTimeout(TimeoutError())
2025-07-09 00:18:02,556 - __main__ - ERROR - Could not send startup message to TELEGRAM_CHAT_ID 7049055170: Timed out
2025-07-09 00:18:02,557 - telegram.ext.ExtBot - DEBUG - Entering: get_me
2025-07-09 00:18:02,557 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
